src/OneFit.WebApi/Controllers/FacilityController.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.AspNetCore.Mvc;
+using OneFit.Service.DTOs.Facilities;
+using OneFit.Service.Services.Facilities;
+using OneFit.WebApi.Models;
+
+namespace OneFit.WebApi.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class FacilityController : ControllerBase
```

Comment:
Name of the controller must be plural. For example, use FacilitiesController instead of FacilityController